src/pages/node/channelsIncoming.tsx

```diff
@@ -127,7 +144,7 @@ function ChannelsPage() {
     return {
       id: channel.id,
       key: key.toString(),
-      peerAddress: channel.peerAddress,
+      peerAddress: getAliasByPeerAddress(channel.peerAddress),
       status: channel.status,
       funds: `${utils.formatEther(channel.balance)} ${HOPR_TOKEN_USED}`,
       actions: (
```

Comment:
We would also want to activate the ping modal again, so i guess getAliasOrPeerIdFromPeerAddress is needed

Reply:
added getPeerIdFromPeerAddress to make the ping work